Makefile

```diff
@@ -40,9 +40,10 @@ coverage.go:
 docs.generate:
 	# Temporarily replaces ~/.rill/config.yaml to avoid including user-defined defaults in generated docs.
 	# Sets version to the latest tag to simulate a production build, where certain commands are hidden.
-	rm -rf docs/docs/reference/cli
+	rm -rf docs/docs/reference/cli docs/docs/reference/project-files
 	if [ -f ~/.rill/config.yaml ]; then mv ~/.rill/config.yaml ~/.rill/config.yaml.tmp; fi;
-	go run -ldflags="-X main.Version=$(shell git describe --tags `git rev-list --tags --max-count=1`)" ./cli docs generate docs/docs/reference/cli/
+	go run -ldflags="-X main.Version=$(shell git describe --tags `git rev-list --tags --max-count=1`)" ./cli docs generate-cli docs/docs/reference/cli/
+	go run -ldflags="-X main.Version=$(shell git describe --tags `git rev-list --tags --max-count=1`)" ./cli docs generate-project --project runtime/parser/schema/project.schema.json --rillyaml runtime/parser/schema/rillyaml.schema.json  --out docs/docs/reference/project-files
```

Comment:
```
--project runtime/parser/schema/project.schema.json --rillyaml runtime/parser/schema/rillyaml.schema.json
```
Can you hard-code the schema paths paths in the command instead of passing here? They are unlikely to change, so better keep the config in the code